R/update.R

```diff
@@ -34,6 +41,29 @@
 #' fit4 <- update(fit1, family = weibull(), init = "0",
 #'                prior = set_prior("normal(0,5)"))
 #' summary(fit4)
+#'
+#' ## to avoid a recompilation when updating a 'cmdstanr'-backend fit in a fresh
+#' ## R session, set option 'cmdstanr_write_stan_file_dir' before creating the
+#' ## initial 'brmsfit'
+#' ## CAUTION: the following code creates some permanent files (two
+#' ## 'model_<hash>.stan' files and one 'model_<hash>(.exe)' executable) in the
+#' ## current working directory, additionally to file 'fit_cmdstanr.rds' (which
+#' ## is deleted afterwards)
+#' if (!file.exists("fit_cmdstanr.rds")) {
```

Comment:
Why do you check for the .rds file in this example?

Reply:
In case the user has such a file lying around in the current working directory (unlikely, but not impossible).

Reply:
Ok, then I would add `as.numeric(Sys.time())`. Is that ok?

Reply:
Yeah, I think

Reply:
Oh, taking the time is a bad idea because the example involves a restart of the R session in between. I'll follow your RNG suggestion.

Reply:
I picked the RNG suggestion now (commit https://github.com/paul-buerkner/brms/pull/1374/commits/89d56d1e3140725619672ba4b1ae4875518e452a).